backends/platform/ios7/ios7_osys_main.h

```diff
@@ -96,8 +96,6 @@ class OSystem_iOS7 : public ModularGraphicsBackend, public EventsBaseBackend {
 	void setFeatureState(Feature f, bool enable) override;
 	bool getFeatureState(Feature f) override;
 
-	bool setGraphicsMode(int mode, uint flags) override;
-
 	TouchMode getCurrentTouchMode() const { return _currentTouchMode; };
```

Comment:
; after functions

Reply:
I prefer not to modify this one to not blur the code changes.
The PR is already big enough. :)